docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -108,6 +116,10 @@ For example:
 = orderCount > 15 or totalPrice > 50
 
 = list contains([6,7], today().weekday)
+
+= numberOfCompletedInstances = 2
+
+= numberOfCompletedInstances/numberOfInstances >= 0.5
```

Comment:
👍  I really like the new additions.

🔧  This list of examples is becoming pretty long which might confuse users. Let's reduce it a bit by removing the expressions that use `orderCount`.